src/DistortableImageOverlay.js

```diff
@@ -534,8 +555,8 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
   },
 });
 
-L.distortableImageOverlay = function(id, options) {
-  return new L.DistortableImageOverlay(id, options);
+L.distortableImageOverlay = function(id, options) { // remove temp
```

Comment:
A couple comments to remove here! Thanks!

Reply:
Yes, I will remove. Thanks!